Library/Formula/zorba.rb

```diff
@@ -0,0 +1,46 @@
+
+class Zorba < Formula
+  homepage "http://www.zorba.io/"
+  url "https://github.com/28msec/zorba/archive/3.0.tar.gz"
+  sha1 "c444cde689600aab1172b4974348dd6626e92261"
+
+  option "disable-big-integer", "Use 64 bit precision instead of arbitrary precision for performance"
```

Comment:
All options should be ` with ` or ` without `, such as ` with-big-interger `, etc.